AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -428,6 +428,7 @@ malaysiastock.biz#%#//scriptlet("set-constant", "google_ad_status", "1")
 ||tsn.ua/src/desktop/img/placeholders/
 tsn.ua##.c-aside--60x35
 ||sp.azcentral.com^
+joyeditor.ru#%#//scriptlet("abort-on-property-read", "mdpDeBlocker")
```

Comment:
I think that it should be moved to Base/Russian filter.
https://github.com/AdguardTeam/AdguardFilters/blob/94c2b447b98e7f49c36d8c45cc4da9579582a29a/EnglishFilter/sections/antiadblock.txt#L30

By the way, it seems that it's already fixed by `/wp-content/plugins/deblocker/js/deblocker.min*.js$~third-party`, at least on my side.

Reply:
> By the way, it seems that it's already fixed by `/wp-content/plugins/deblocker/js/deblocker.min*.js$~third-party`, at least on my side.

I have the same. Maybe this rule doesn't apply for some unknow reason. Add `joyeditor.ru` to the common scriptlet rule in the `mdpDeBlocker` section (AdGuard Base).